src/main/java/io/swagger/codegen/v3/generators/DefaultCodegenConfig.java

```diff
@@ -4289,4 +4299,8 @@ public ISchemaHandler getSchemaHandler() {
     public OpenAPI getOpenAPI() {
         return this.openAPI;
     }
+
+    public boolean isNotNullJacksonAnnotation() {
+        return notNullJacksonAnnotation;
+    }
```

Comment:
this is specific to a generator, not to the whole set, DefaultCodegen must be left unchanged, you would code this into the Java generator

Reply:
done